.github/workflows/release.yaml

```diff
@@ -17,10 +17,10 @@ jobs:
       - name: Setup .NET
         uses: actions/setup-dotnet@v3
         with:
-          dotnet-version: 6.0.404
-          
+          dotnet-version: 6.0.406
+
       - name: Ensure that dotnet version is used
-        run: dotnet new globaljson --sdk-version 6.0.404 --roll-forward latestFeature
+        run: dotnet new globaljson --sdk-version 6.0.406 --roll-forward latestFeature
```

Comment:
I think this line is redundant if we already have a global.json file

Reply:
Yes, I see it exactly as you do.
I am in the process of adjusting this.